src/chainparams.cpp

```diff
@@ -194,34 +194,77 @@ class CMainParams : public CChainParams
+        checkpointData.mapCheckpoints.emplace(
+            134444, uint256S("0x00000000000005b12ffd4cd315cd34ffd4a594f430ac814c91184a0d42d2b0fe"));
+        checkpointData.mapCheckpoints.emplace(
+            168000, uint256S("0x000000000000099e61ea72015e79632f216fe6cb33d7899acb35b75c8303b763"));
+        checkpointData.mapCheckpoints.emplace(
+            193000, uint256S("0x000000000000059f452a5f7340de6682a977387c17010ff6e6c3bd83ca8b1317"));
+        checkpointData.mapCheckpoints.emplace(
+            210000, uint256S("0x000000000000048b95347e83192f69cf0366076336c639f9b7228e9ba171342e"));
+        checkpointData.mapCheckpoints.emplace(
+            216116, uint256S("0x00000000000001b4f4b433e81ee46494af945cf96014816a4e2370f11b23df4e"));
+        checkpointData.mapCheckpoints.emplace(
+            225430, uint256S("0x00000000000001c108384350f74090433e7fcf79a606b8e797f065b130575932"));
+        checkpointData.mapCheckpoints.emplace(
+            250000, uint256S("0x000000000000003887df1f29024b06fc2200b55f8af8f35453d7be294df2d214"));
+        checkpointData.mapCheckpoints.emplace(
+            279000, uint256S("0x0000000000000001ae8c72a0b0c301f67e3afca10e819efa9041e458e9bd7e40"));
+        checkpointData.mapCheckpoints.emplace(
+            295000, uint256S("0x00000000000000004d9b4ef50f0f9d686fd69db2e03af35a100370c64632a983"));
+        // August 1st 2017 CASH fork (UAHF)
+        checkpointData.mapCheckpoints.emplace(
+            478559, uint256S("0x000000000000000000651ef99cb9fcbe0dadde1d424bd9f15ff20136191a5eec"));
+        // November 13th 2017 new DAA fork
+        checkpointData.mapCheckpoints.emplace(
+            504031, uint256S("0x0000000000000000011ebf65b60d0a3de80b8175be709d653b4c1a1beeb6ab9c"));
+        // May 15th 2018 re-enable op_codes and 32 MB max block size
+        checkpointData.mapCheckpoints.emplace(
+            530359, uint256S("0x0000000000000000011ada8bd08f46074f44a8f155396f43e38acf9501c49103"));
+        // add checkpoint on non BSV fork only
+        if (nMiningSvForkTime == 0)
+        {
+            // Nov 15th 2018 activate LTOR, DSV op_code
+            checkpointData.mapCheckpoints.emplace(
+                556767, uint256S("0000000000000000004626ff6e3b936941d341c5932ece4357eeccac44e6d56c"));
+            checkpointData.nTimeLastCheckpoint = 1542304936; // * UNIX timestamp of last checkpoint block
+            // * total number of transactions between genesis and last checkpoint
+            checkpointData.nTransactionsLastCheckpoint = 265567564;
+            // * estimated number of transactions per day after checkpoint (~3.5 TPS)
+            checkpointData.fTransactionsPerDay = 280000.0;
+        }
+        else if (nMiningSvForkTime != 0)
+        {
+            // Nov 15th 2018 SV fork, 128MB blocks, re-enable bitcoin 0.1.0 op_codes
+            checkpointData.mapCheckpoints.emplace(
+                556767, uint256S("000000000000000001d956714215d96ffc00e0afda4cd0a96c96f8d802b1662b"));
```

Comment:
Height and hash for both SV and CASH fork point are correct.

Adding the same checkpoint for testnet could be done in a subsequent PR.

Reply:
to comment on the last few things u said here, i could change to @Justaphfs method if people prefer, it doesnt matter to me since there is only a minor difference in what the two do.

i can tack on the testnet checkpoint to this PR if u want to pm me the info for it

Reply:
I think @Justaphf's syntax is more readable, but it;s just an aesthetic preference rather than anything else. 

We could postpone the addition testnet checkpoint to a separate PR.